pytorch_lightning/loops/fit_loop.py

```diff
@@ -171,8 +172,23 @@ def run(self) -> None:
         if not self._should_skip_training():
             return super().run()
 
+    def _create_progress(self):
+        # create progress tracker
+        # the deepcopy is used to avoid sharing same reference for each optimizer.
+
+        # todo: (tchaton) Temporary check to be removed when progress are added to checkpoint.
+        if getattr(self.training_loop, "progress", None) is None:
+            self.training_loop.progress = TrainingLoopProgress(num_optimizers=len(self.trainer.optimizers))
+
+        if getattr(self, "progress", None) is None:
+            self.progress = FitLoopProgress(train=self.training_loop.progress)
+
+        self.progress.train.epoch.current.reset()
+
     def on_run_start(self) -> None:
         """Calls the ``on_train_start`` hook."""
+        self._create_progress()
```

Comment:
maybe move this to the `connect` function?

Reply:
This won't work as we need the `optimizers` and `lr_schdulers` to be properly created.